commons/src/main/scala/com/lynbrookrobotics/potassium/commons/drivetrain/UnicycleDrive.scala

```diff
@@ -9,6 +9,7 @@ trait UnicycleProperties {
   val maxForwardVelocity: Velocity
   val maxTurnVelocity: AngularVelocity
   val maxAcceleration: Acceleration
+  val initPosition: Point
```

Comment:
I was thinking that having an init position would be convenient because we might have different starting configurations in auto, so if we change our initial position, we only have to change this  opposed to changing how for we drive relatively for every auto task we use.